widget/contrib/gpmdp.lua

```diff
@@ -6,100 +6,97 @@
                                                      
 --]]
 
-local helpers             = require("lain.helpers")
-local json                = require("lain.util.dkjson")
-local watch               = require("awful.widget").watch
-local focused             = require("awful.screen").focused
-local naughty             = require("naughty")
-local next, getenv, table = next, os.getenv, table
+local naughty = require("naughty")
```

Comment:
ah, i thought it was a lua thing to line up the assignment operators, monkey see monkey do :+1: